### ISSUE TITLE
Incorrect installation instructions

### ISSUE THREAD

**Issue by AlexWayfer** [RensAlthuis/vertical-overview#11]
Hello.

I've done `git clone`, `cd`, but `make` returns an error:

```
> make
rm -r _build
rm: cannot remove '_build': No such file or directory
make: *** [Makefile:8: clean] Error 1
```

What I should to do? And should we change the README?

**Comment by RensAlthuis** [comment]
My bad, I should have tested more carefully. This commit should fix it a329c6d, could you check to see if it works?

**Comment by AlexWayfer** [comment]
Thank you, it helped. I'm going to create a PR with additional improvements and it'll close this issue completely, I think.
